dev/src/sdlaudiosfxr.cpp

```diff
@@ -417,18 +417,85 @@ void SDLSoundClose() {
     SDL_QuitSubSystem(SDL_INIT_AUDIO);
 }
 
-bool SDLPlaySound(string_view filename, bool sfxr, int vol) {
+int SDLPlaySound(string_view filename, bool sfxr, float vol, int loops) {
     #ifdef __EMSCRIPTEN__
     // Distorted in firefox and no audio at all in chrome, disable for now.
-    return false;
+        return 0;
     #endif
 
     if (!SDLSoundInit())
-        return false;
+        return 0;
 
     auto snd = LoadSound(filename, sfxr);
+    int ch = -1;
     if (snd) {
```

Comment:
just do an `if (snd) return 0`, that way you don't have to start `ch` at -1, and it is all easier to read.

Reply:
Ok